src/main/java/net/mcreator/ui/workspace/breadcrumb/WorkspaceFolderBreadcrumb.java

```diff
@@ -185,4 +168,24 @@ public interface SelectionListener {
 		void elementSelected(IElement element, JComponent component, MouseEvent event);
 	}
 
+	public static class Small extends WorkspaceFolderBreadcrumb {
+
+		public Small(MCreator mcreator) {
+			super(mcreator, 3, true);
+			reloadPath(mcreator.mv.currentFolder, FolderElement.class);
+			setSelectionListener((element, component, event) -> {
+				if (element instanceof FolderElement fe)
+					reloadPath(fe, FolderElement.class);
+			});
+		}
+
+		public JScrollPane getInScrollPane() {

```

Comment:
Fine to add this in the inner class?

Reply:
Yes, this was the idea so this looks good to me